docs/index.md

```diff
@@ -3,12 +3,13 @@ title: Windows Community Toolkit Documentation
 author: nmetulev
 ms.topic: overview
 description: The Windows Community Toolkit is a collection of helper functions, custom controls, and app services. It simplifies and demonstrates common developer tasks building UWP apps for Windows 10. 
-keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit
-ms.custom: RS5
 ---
 
 # Windows Community Toolkit Documentation
 
+> [!IMPORTANT]
+> This Windows Community Toolkit is moving to an archived state. See the updated [Windows Community Toolkit documentation](/dotnet/communitytoolkit/windows/) in the .NET guidance.
```

Comment:
@mattwojo I would maybe change this to "This Windows Community Toolkit **documentation** is moving to an archived state." I don't know if you want to make statements about the status of the toolkit itself. But I don't know much about it, so just a thought.

Reply:
Great catch! Updated.